R4ML/R/ml.model.svm.R

```diff
@@ -150,24 +151,24 @@ setMethod("initialize", "r4ml.svm",
 #' @examples \dontrun{
 #' 
 #' # Load the Iris dataset 
-#' iris_hf <- as.r4ml.frame(iris)
+#' iris_r4mlDF <- as.r4ml.frame(iris)
```

Comment:
I'm not sure what the best answer is here. I think ideally the name should all be lowercase. I've been using `r4_...`